shopping-cart/shopping-render.js

```diff
@@ -0,0 +1,50 @@
+import { calcLineItem } from '../findID_calc_functions/utils.js';
+export default function renderlineitem(Angeladrinks, cartItem) {
```

Comment:
Use at most one line of whitespace between pieces of the function. That makes it more readable